src/evd/evd.cpp

```diff
@@ -70,6 +70,25 @@ int evd_process(evdOptions *opts)
     std::cout << "Number of bands = " << nbands << "\n";
 
 
+    //Resolve method and bandwidth
+    if (opts->method.compare("SBAS") == 0)
+    {
+        //If bandwidth is not provided
+        if (opts->bandWidth <= 0)
+        {
+            opts->method = "EVD";
+            std::cout << "Since SBAS bandwidth was not provided - using EVD or full bandwidth \n";
+        }
```

Comment:
If user has asked for "SBAS" method, and some parameters are missing, I don't think if we should silently change users requested method to "EVD". In this case, I suggest that the workflow throw an error message and exist. Otherwise I'm afraid this will be a cause of confusion.

Reply:
Now use STBAS everywhere consistently

Reply:
And throw an error if bandwidth not provided with STBAS